packages/orchestration/src/exos/orchestrator.js

```diff
@@ -39,6 +40,11 @@ export const OrchestratorI = M.interface('Orchestrator', {
   getChain: M.call(M.string()).returns(Vow$(ChainInfoShape)),
   makeLocalAccount: M.call().returns(Vow$(LocalChainAccountShape)),
   getBrandInfo: M.call(DenomShape).returns(BrandInfoShape),
+  getDenomOn: M.call(
+    DenomShape,
+    M.remotable('Chain'),
```

Comment:
it is remotable but it lives in the same vat. can we have a stricter type in that case? (curious, not blocking)

Reply:
no; patterns are pass-invariant, and "in the same vat" is the antithesis of pass-invariant.